ast/cinaps/grammar.ml

```diff
@@ -1,24 +1,37 @@
 open Stdppx
 
-let rec string_of_ty ~internal ty =
+let string_of_name ~internal name =
+  if internal then Ml.id name else Ml.module_name name ^ ".t"
+
+let string_of_var ?nodify var =
+  match nodify with
+  | None -> Ml.tvar var
+  | Some () -> Ml.poly_inst "node" ~args:[Ml.tvar var]
+
+let string_of_targ ?nodify ~internal targ =
```

Comment:
For purely internal functions `string_of_var` and `string_of_targ`, if we make `nodify` a required argument, we can never forget to pass it in.